code/game/atoms_movable.dm

```diff
@@ -36,6 +36,12 @@
 
 	var/datum/component/orbiter/orbiting
 
+	/** General use assoc list for cooldowns.
+		Example usage
+		cooldowns[COOLDOWN_CHEW] = addtimer(VARSET_LIST_CALLBACK(cooldowns, COOLDOWN_CHEW, null), 7.5 SECONDS)
+	 */
+	var/list/cooldowns = list()
```

Comment:
Does this documentation work as intended?

Reply:
It did, but it wasn't clear (it was a single line)
![image](https://user-images.githubusercontent.com/1134201/80543659-4e095680-89a7-11ea-8485-29bb3f6ec00e.png)


I've updated it again